code/game/objects/items/weapons/storage/deferred.dm

```diff
@@ -318,3 +318,50 @@
 	/obj/item/clothing/suit/armor/greatcoat/german_overcoat = 1,
 	/obj/item/clothing/under/germansuit = 1)
 
+/obj/item/weapon/storage/deferred/crate/clown_crime
```

Comment:
Have some fucking mercy for people who will spawn these

Reply:
1984
gimme a second

Reply:
changed it all